config/prod/tom-test-rstudio4.yaml

```diff
@@ -0,0 +1,46 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-linux-v2.j2
+stack_name: tom-test-rstudio2
+sceptre_user_data:
+  Distro: "ubuntu"     # (valid values: aws, ubuntu) AMIId must match distro
+  # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
+  # OpenPorts: ["80","443"]
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "Platform"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infrastructure"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "sandcastlevpc"
+  KeyName: "sandbox"
+  # AMIId: "ami-0810a318c4b1243c5"      # https://github.com/Sage-Bionetworks-IT/packer-base-amazonlinux2/releases/tag/v1.0.2
+  AMIId: "ami-0cd489a193a7cc7de"    # https://github.com/Sage-Bionetworks-IT/packer-rstudio/releases/tag/v1.0.2
```

Comment:
this isn't v1.0.3, the v1.0.3 tagged build reports `ami-01866cea595e4f968`
https://travis-ci.com/github/Sage-Bionetworks-IT/packer-rstudio/jobs/319496371#L442